src/videoCallWindow/preload/index.ts

```
@@ -10,8 +15,15 @@ contextBridge.exposeInMainWorld('videoCallWindow', {
     return new Promise<string | null>((resolve) => {
       ipcRenderer.once(
         'video-call-window/screen-sharing-source-responded',
-        (_event, id) => {
-          resolve(id);
+        (
+          _event,
+          payload: string | null | ScreenSharingSelectionPayload
+        ) => {
```

ESLint

[error] Replace `⏎··········_event,⏎··········payload:·string·|·null·|·ScreenSharingSelectionPayload⏎········` with `_event,·payload:·string·|·null·|·ScreenSharingSelectionPayload`

(prettier/prettier)